scripts/shared.js

```diff
@@ -1148,7 +1148,8 @@ export function buildArticleCardsBlock(count, templateName, addBlockToDom) {
  *  placeholders.
  */
 export function loadTemplateArticleCards(main, templateName, articles) {
-  const placeholderCards = main.querySelectorAll(`.article-cards[data-template="${templateName}"] .article-card.skeleton`);
+  const placeholderCards = main.querySelectorAll((`.article-cards[data-template="${templateName}"] .article-card.skeleton`)
+  || (`.article-cards[data-template="${templateName}"] .article-card`));
```

Comment:
I don't think we need this change, so we'll be loading content into article card skeletons?

Reply:
ah left that there from previous implementation. removed